CMakeLists.txt

```diff
@@ -1,39 +1,20 @@
 cmake_minimum_required(VERSION 3.0)
 
-option(WARN_VERBOSE "Enable highest warning level and disable warnings as errors" OFF)
+project(shaderunner)
```

Comment:
I kept a single project instead of multiple ones, feel free to revert if that is not intended